### PR TITLE
Shopfloor-to-Cloud Update

### PR DESCRIPTION
**Pull request by RCAutomatizacion**
Please leave your review...

### CODE REVIEW

File: docs/graphics/architecture3.png
Comment (Boehmi-TB) [inline docs/graphics/architecture3.png:1]: Remove IED -> IEM connection Port 32500.
IEM Pro, Cloud, IEM V also use port 443 for remote access.

Comment (Boehmi-TB) [reply docs/graphics/architecture3.png:1]: IED Southbound Connectivity to IED Northbound is via port 9884 not 9883 please adjust

Comment (RCAutomatizacion) [reply docs/graphics/architecture3.png:1]: Done. Thanks for feedback.

**Comment by RCAutomatizacion** [comment]
I fixed the merge conflicts. I did the suggested changes. Please check and give feedback. If everything is ok, approve pull request.

File: README.md [hunk 1]
```diff
@@ -67,19 +69,24 @@ TIA & PLCs:
...
+- IIH Essentials V1.10.0
+- Flow Creator V1.17.0-2
+- Energy Manager V1.16.1
+- DataXess V1.4.1-3
+- IIH Semantics V1.9.0-0
+- Common Configurator V1.9.0-4
 - Web browser (Mozilla or Chrome)
 
+- Common Connector Configurator V1.9.1-1
+- DataXess Configurator V1.4.2-4
```
Comment (Boehmi-TB) [inline README.md:88]: Fix spacing of list.
Used Components and Configuration North and Southbound spacing is different to Prerequisites
image

File: docs/graphics/CommonConfigurator_4.png
Comment (Boehmi-TB) [inline docs/graphics/CommonConfigurator_4.png:1]: Here is port 502 configured in Screenshot. Default Port for OPC UA Server which is used by S7-1500 PLC as well is port 4840

File: docs/install_PLC_Devices_Southbound.md [hunk 2]
```diff
@@ -20,177 +23,163 @@
...
-    - Metadata: `ie/m/j/simatic/v1/energy1line1:iefc/dp`
-    - Data: `ie/d/j/simatic/v1/energy1line1:iefc/dp/r/line1/default`
     
-    Energy2:
-    - Metadata: `ie/m/j/simatic/v1/energy2line2:iefc/dp`
-    - Data: `ie/d/j/simatic/v1/energy2line2:iefc/dp/r/line2/default`
+  ![FlowCreator1](graphics/Flow_Creator1.png)
+
+2. Double click on a MQTT-Node  
+3. Add IE Databus Credentials
```
Comment (Boehmi-TB) [inline docs/install_PLC_Devices_Southbound.md:168]: IE Databus -> Databus

File: docs/install_PLC_Devices_Southbound.md [hunk 3]
```diff
@@ -20,177 +23,163 @@
...
+  ![FlowCreator3](graphics/Flow_Creator3.png)
 
-6. Mark the data and metadata routes an click "Save Route" 
-      
-        
-  ![Cloud_Connector7](graphics/Cloud_Connector_Route1.png)
-    
-      
-  ![Cloud_Connector8](graphics/Cloud_Connector_Route2.png)
+Before deploying the flows, the following topic has to be changed to `ie/d/j/simatic/v1/opcuac1/dp/r/Energy1/default` for Energy 1 and to `ie/d/j/simatic/v1/opcuac1/dp/r/Energy2/default` for Energy 2:
```
Comment (Boehmi-TB) [inline docs/install_PLC_Devices_Southbound.md:175]: Can you update the two flow creator json that this step is not required.

File: docs/install_Device_Northbound.md [hunk 4]
```diff
@@ -2,27 +2,27 @@
...
     - [Configure Connections](#configure-connections)
     - [Create the Asset Model](#create-the-asset-model)
   - [Northbound Device - Energy Manager](#northbound-device---energy-manager)
 - [Navigation](#navigation)
   
 # Configure Northbound
 
-The Northbound consist of one device. In the following this is called "Central Device".
+The Northbound consist of one device. In the following this is called "Shopfloor-to-Cloud".
 Installed Apps on Central Device:
```
Comment (Boehmi-TB) [inline docs/install_Device_Northbound.md:17]: Installed Apps are:

As you are not using Central Device anymore but renamed it, remove it from here

File: docs/install_Device_Northbound.md [hunk 5]
```diff
@@ -2,27 +2,27 @@
...
-  - Data Service
+  - IIH Essentials
   - Databus
-  - Energy Manager
-  - External Databus  
-  - IIH Core
-  - IIH Configurator
+  - DataXess 
+  - IIH Semmantics
```
Comment (Boehmi-TB) [inline docs/install_Device_Northbound.md:21]: IIH Semantics

File: docs/install_Device_Northbound.md [hunk 6]
```diff
@@ -44,96 +44,94 @@ Instead of manually configuring you can also import the configuration files:
...
 
 3. Confirm the integration in the popup window
 
-  ![IIH_IntegrateDataServiceConfirmation](graphics/IIH_IntegrateDataServiceConfirmation.png)
+  ![IIH_IntegrateConfirmation](graphics/IIH2.png)
+
+### Configure Connection to Insights Hub.
 
-### Configure Connections
+Common Configurator needs a connection to the Databus and to Insights Hub.
```
Comment (Boehmi-TB) [inline docs/install_Device_Northbound.md:108]: Especially IIH Essentials needs a connection to Databus for colecting data

File: docs/install_Device_Northbound.md [hunk 7]
```diff
@@ -44,96 +44,94 @@ Instead of manually configuring you can also import the configuration files:
...
+4. Select the Target type. Enter credentials for the application. Those are required, to allow the IIH to interact with Insights Hubs REST-API.
+   
+   ![IIH_InsightsHub](graphics/IIH5.png)
 
-  ![IIH_MindsphereConfig2](graphics/IIH_MindsphereConfig2.png)
+   The Insights Hub Tenant Administrator has to create them and assign the role "mdsp:core:Admin3rdPartyTechUser", to allow the IIH to update the asset model in Insights Hub. More information can be found in the [IIH documentation](https://cache.industry.siemens.com/dl/dl-media/582/109803582/att_1087779/v6/EdgeApp_CommonConfigurator_en-US/en-US/index.html#treeId=487dc5ea471ae3fbdfb56d715301dad7) and [here](https://documentation.mindsphere.io/MindSphere/apps/operator-cockpit/application-credentials-for-API-applications.html)
```
Comment (Boehmi-TB) [inline docs/install_Device_Northbound.md:126]: I would recommend not using Application Credentials with Admin3rdPartyTechUser but create technical user in Settings of Insights Hub as this is now the prefered method to authenticate IIH against REST API. Currently not part of Doku

File: docs/install_Device_Northbound.md [hunk 8]
```diff
@@ -44,96 +44,94 @@ Instead of manually configuring you can also import the configuration files:
...
 
-7. Enter the certificate details and upload the certificate and key. Information on how to create connector certificates
-   can be found [here](https://documentation.mindsphere.io/MindSphere/howto/howto-onboard-mindconnect-mqtt.html) and [here](https://documentation.mindsphere.io/MindSphere/howto/howto-managing-ca-certificates.html)
+   Enter the certificate details and upload the certificate and key. Information on how to create connector certificates can be found [here](https://documentation.mindsphere.io/MindSphere/howto/howto-onboard-mindconnect-mqtt.html) and [here](https://documentation.mindsphere.io/MindSphere/howto/howto-managing-ca-certificates.html)
```
Comment (Boehmi-TB) [inline docs/install_Device_Northbound.md:128]: For user with least know how the prefered option to create certificates for MQTT can be found here:
https://documentation.mindsphere.io/MindSphere/howto/howto-obtaining-auto-generated-agent-certificate.html

File: docs/install_Device_Northbound.md [hunk 9]
```diff
@@ -143,165 +141,36 @@ certificates to allow a connection from the IIH.
 
 3. Add two Aspects to structure the data according two our two machines
 
-  ![IIH_CreateAssetModel1](graphics/IIH_CreateAssetModel1.png)
+  ![IIH_CreateAssetModel1](graphics/IIH7.png)
```
Comment (Boehmi-TB) [inline docs/install_Device_Northbound.md:144]: I would change the Asset Model as the definition for what is an Asset and what is an Aspect is not followed.

Asset is a representation of a physical system like a machine or automation system. (Device, Robot, Production Line, etc.)

Aspects models information for example on a use case basis (e.g. Energy Management) which can be attached to various assets which e.g. have energy consumptions.

Line1 (Asset)
- Media Consumption (Aspect)
  - totalEnergy
  - totalWater
  - ...
Line2 (Asset)
- Media Consumption (Aspect)
  - totalEnergy
  - totalWater
  - ...
  
Also don't name "Line1_TotalEnergy" but used name TotalEnergy as it is already grouped underneath Line1

File: README.md (hunk 1 above)
Comment (RCAutomatizacion) [reply README.md:86]: Done

File: docs/install_Device_Northbound.md (hunk 4 above)
Comment (RCAutomatizacion) [reply docs/install_Device_Northbound.md:17]: Done

File: docs/install_Device_Northbound.md (hunk 6 above)
Comment (RCAutomatizacion) [reply docs/install_Device_Northbound.md:108]: The point of that statement is to say that Common Configurator (with IIH Essentials integrated as shown in the previos part) need Databus and Insights Hub.

File: docs/install_Device_Northbound.md (hunk 8 above)
Comment (RCAutomatizacion) [reply docs/install_Device_Northbound.md:128]: I added it, done

File: docs/install_PLC_Devices_Southbound.md (hunk 3 above)
Comment (RCAutomatizacion) [reply docs/install_PLC_Devices_Southbound.md:175]: Done

**Comment by RCAutomatizacion** [comment]
I just finished the suggested modifications, please let me know your feedback.

File: docs/graphics/IHUI6.png
Comment (Boehmi-TB) [inline docs/graphics/IHUI6.png:1]: Username and password visible

File: docs/install_Device_Northbound.md [hunk 10]
```diff
@@ -44,264 +43,224 @@ Instead of manually configuring you can also import the configuration files:
...
+    ![IHUI4](graphics/IHUI4.png)
 
-6. Assign those tags two the variables in your Asset model per drag & drop
+    The following tab will appear. Make sure to save the generated password and assign the previously described roles:
 
-  ![IIH_CreateAssetModel2](graphics/IIH_CreateAssetModel2.png)
+    ![IHUI6](graphics/IHUI6.png)
 
-7. Check the "Storage" and "Cloud Sync" checkbox for each variable
+    And that's it. Now assign the required information as follows:
```
Comment (Boehmi-TB) [inline docs/install_Device_Northbound.md:163]: Remove "And that's it"

**Comment by Boehmi-TB** [comment]
Export Energy Manager Dashboards from Insights Hub and add them to repository

File: docs/install_Device_Northbound.md (hunk 10 above)
Comment (RCAutomatizacion) [reply docs/install_Device_Northbound.md:163]: Done

**Comment by RCAutomatizacion** [comment]
I just finished the suggested modifications, please let me know your feedback.

**Review by Boehmi-TB — APPROVED** [review]

**Pull request merged**